tests/unit/test_utils.py

```diff
@@ -438,3 +440,123 @@ def test_FedoraDistGit(tmpdir):
     fedora_sources_obj = tmt.utils.FedoraDistGit()
     assert [("https://src.fedoraproject.org/repo/pkgs/rpms/tmt/fn-1.tar.gz/sha512/09af/fn-1.tar.gz",
             "fn-1.tar.gz")] == fedora_sources_obj.url_and_name(cwd=tmpdir)
+
+
+class Test_validate_git_status:
+    def test_all_good(cls, tmpdir):
+        pass
+
+    def test_no_remote(cls, tmpdir):
+        subprocess.run('git init'.split(), check=True, cwd=tmpdir)
```

Comment:
Here also a `run()` function would make the code more readable. See also my comment in `tmt.utils`.